weblate_web/templates/user.html

```diff
@@ -14,47 +17,55 @@ <h1 class="section-title min-m">{% trans "User profile" %}</h1>
             </div>
             {% with services=user.service_set.all %}
             {% if services %}
-            <h1>{% trans "Your services" %}</h1>
-            {% for service in services %}
+            <div class="updated-service-title">
+                <h1>{% trans "My services" %}<span><a href="">{% trans "See user profile " %}<img src="{% static "img/ArrowR.png" %}" alt=">" /></a></span></h1>
```

Comment:
It will point to Hosted Weblate:

```suggestion
                <h1>{% trans "My services" %}<span><a href="https://hosted.weblate.org/accounts/profile/">{% trans "See user profile" %}<img src="{% static "img/ArrowR.png" %}" alt=">" /></a></span></h1>
```

Reply:
Done!